### DISCUSSION TITLE
Speculation on the cause of the stagnation of thorium construction

### DISCUSSION THREAD

**Discussion by whxuezhe** [Alex313031/Thorium-Win-AVX2#26] [General]
Some impacts caused by the epidemic situation or the global climate and energy crisis, as well as the bugs that cannot be solved in a short time during the next build of m106.0.5199.0, so that the new thorium build may be pushed only when the combined development workflow is pushed to m107.0.

**Comment by gz83** [comment]
> Some impacts caused by the epidemic situation or the global climate and energy crisis, as well as the bugs that cannot be solved in a short time during the next build of m106.0.5199.0, so that the new thorium build may be pushed only when the combined development workflow is pushed to m107.0.

@whxuezhe The new AVX2 version will be released soon, please pay attention to the dynamics of the repo.

By the way, NTP page crashes and settings page crashes have been fixed. In addition, the recent changes made in the main repo will be introduced into the new version, and the new version will introduce polly optimization at the compilation level.

You can learn about polly here, and this page will also show other optimizations used by thorium: https://thorium.rocks/optimizations

**Comment by Alex313031** [comment]
@whxuezhe Yessir, ^

@gz83 Thanks for responding to this guy
